dev-lang/hy/hy-0.19.0.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
 RESTRICT="test" # needs some pointy sticks. Seriously.
```

Comment:
Can we fix tests?

Reply:
Looks like you did, thank you!

Reply:
Fixed, now they run wih "test" use flag. But single test is failing, figuring out why. Just running them locally works well, but when they run by portage - one macro case fails.

Reply:
Network or similar? We can skip fragile tests like that, but if we can figure it out, that's obviously nicer.

Reply:
Looks like something is misplaced within build directory during tests run. I commented out failing assert and next failed too. Maybe there are some docs I didn't notice that can help me with debugging tests? I call `ebuild <file> test` but cannot run single case. Is there the way to pass args to pytests via `ebuild` command or I have to run tests somehow from portage work directory?